### PR TITLE
Service accounts: doc fixes

### PR DESCRIPTION
**Pull request by IevaVasiljeva**
**What this PR does / why we need it**:
Clarifies service account documentation to make this feature easier to use.

**Which issue(s) this PR fixes**:

Related to https://github.com/grafana/support-escalations/issues/2454

### CODE REVIEW

**Comment by IevaVasiljeva** [comment]
Looks like @mjseaman has already fixed the main confusion in https://github.com/grafana/grafana/pull/48515, but I'll leave this up anyway, as I think it still improves the docs a little bit.

**Review by Jguer — APPROVED** [review]

File: docs/sources/administration/service-accounts/add-service-account-token.md [hunk 1]
```diff
@@ -14,12 +14,11 @@ You can create a service account token using the Grafana UI or via the API. For
 ## Before you begin
 
 - Ensure you have added the `serviceAccounts` feature toggle to Grafana. For more information about adding the feature toggle, refer to [Enable service accounts]({{< relref "./enable-service-accounts.md#">}}).
-- Ensure you have permission to create and edit service accounts. For more information about user roles, refer to [About users and permissions]({{< relref "../manage-users-and-permissions/about-users-and-permissions.md#">}}).
-- [Create a service account in Grafana]({{< relref "./create-service-account.md#">}}).
+- Ensure you have permission to create and edit service accounts. By default, organization administrator role is required to create and edit service accounts. For more information about user permissions, refer to [About users and permissions]({{< relref "../manage-users-and-permissions/about-users-and-permissions.md#">}}).
```
Comment (achatterjee-grafana) [inline docs/sources/administration/service-accounts/add-service-account-token.md:17]:
```suggestion
- Ensure you have permission to create and edit service accounts. By default, the organization administrator role is required to create and edit service accounts. For more information about user permissions, refer to [About users and permissions]({{< relref "../manage-users-and-permissions/about-users-and-permissions.md#">}}).
```

File: docs/sources/administration/service-accounts/add-service-account-token.md [hunk 2]
```diff
@@ -14,12 +14,11 @@ You can create a service account token using the Grafana UI or via the API. For
...
 
-**To add a token to a service account:**
+## To add a token to a service account
 
-1. Sign in to Grafana and hover your cursor over the organization icon in the sidebar.
+1. Sign in to Grafana and hover your cursor over **Configuration** (the gear icon) in the sidebar.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/service-accounts/add-service-account-token.md:21]:
```suggestion
1. Sign in to Grafana, then hover your cursor over **Configuration** (the gear icon) in the sidebar.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: docs/sources/administration/service-accounts/create-service-account.md [hunk 3]
```diff
@@ -16,9 +16,9 @@ For more information about creating service accounts via the API, refer to [Crea
 ## Before you begin
 
 - Ensure you have added the feature toggle for service accounts `serviceAccounts`. For more information about adding the feature toggle, refer to [Enable service accounts]({{< relref "./enable-service-accounts.md#">}}).
-- Ensure you have permission to create and edit service accounts. For more information about user permissions, refer to [About users and permissions]({{< relref "../manage-users-and-permissions/about-users-and-permissions.md#">}}).
+- Ensure you have permission to create and edit service accounts. By default, organization administrator role is required to create and edit service accounts. For more information about user permissions, refer to [About users and permissions]({{< relref "../manage-users-and-permissions/about-users-and-permissions.md#">}}).
```
Comment (achatterjee-grafana) [inline docs/sources/administration/service-accounts/create-service-account.md:19]: Pls make the same change as in the other file.

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**